pytorch_toolkit/object_detection/vehicle-detection/vehicle-detection-0200/config.py

```diff
@@ -0,0 +1,160 @@
+# model settings
+input_size = 256
+image_width, image_height = input_size, input_size
+width_mult = 1.0
+model = dict(
+    type='SingleStageDetector',
+    backbone=dict(
+        type='mobilenetv2_w1',
+        out_indices=(4, 5),
+        frozen_stages=-1,
+        norm_eval=False,
+        pretrained=True
+    ),
+    neck=None,
+    bbox_head=dict(
+        type='SSDHead',
+        num_classes=1,
+        in_channels=(int(width_mult * 96), int(width_mult * 320)),
+        anchor_generator=dict(
+            type='SSDAnchorGeneratorClustered',
+            strides=(16, 32),
+            widths=[
+                [image_width * x for x in
+                 [0.03190665436176343, 0.07246889234100704, 0.13534887659200423,
+                  0.15785373692633903]],
+                [image_width * x for x in
+                 [0.29641301747833876, 0.3296121909277462, 0.2130097277660658, 0.45965227145427356,
+                  0.6790618020935402]],
+
+            ],
+            heights=[
+                [image_height * x for x in
+                 [0.03836520511012714, 0.08658721963691468, 0.15106894690367778,
+                  0.30385265971706454]],
+                [image_height * x for x in
+                 [0.2010092421878013, 0.4087630480186569, 0.6694307467997506, 0.4766551262641776,
+                  0.6573119829938249]],
+            ],
+        ),
+        bbox_coder=dict(
+            type='DeltaXYWHBBoxCoder',
+            target_means=(.0, .0, .0, .0),
+            target_stds=(0.1, 0.1, 0.2, 0.2), ),
+        depthwise_heads=True,
+        depthwise_heads_activations='relu',
+        loss_balancing=True))
+cudnn_benchmark = True
+train_cfg = dict(
+    assigner=dict(
+        type='MaxIoUAssigner',
+        pos_iou_thr=0.4,
+        neg_iou_thr=0.4,
+        min_pos_iou=0.,
+        ignore_iof_thr=-1,
+        gt_max_assign_all=False),
+    smoothl1_beta=1.,
+    use_giou=False,
+    use_focal=False,
+    allowed_border=-1,
+    pos_weight=-1,
+    neg_pos_ratio=3,
+    debug=False)
+test_cfg = dict(
+    nms=dict(type='nms', iou_thr=0.45),
+    min_bbox_size=0,
+    score_thr=0.02,
+    max_per_img=200)
+# model training and testing settings
+# dataset settings
+dataset_type = 'CocoDataset'
+data_root = 'TBD'
```

Comment:
What does that mean?

Reply:
There was no any dataset suitable for this task. I've added instructions how to filter `car` category from MS COCO dataset and updated corresponding fields in configuration files.